src/app/app.component.ts

```diff
@@ -54,10 +54,19 @@ export class MyApp {
 					this.login.password = val;
 				}, (error) => {
 					this.login.password = '';
+				}),
+				this.storage.get('staging').then((val) => {
+					if(val){
+						this.endpoint = 'https://staging.timmerdorp.com/wp-json';
+					} else{
+						this.endpoint = 'https://shop.timmerdorp.com/wp-json';
+					}
+				}, (error) => {
+					this.endpoint = 'https://shop.timmerdorp.com/wp-json';
 				})
 			]).then(() => {
 				this.splashScreen.hide();
-				if (cordova) {
+				if (this.platform.is('cordova')) {
```

Comment:
ahh ja goeie dat werkt beter... in normaal javascript zou mijn manier ook gewerkt hebben